starter_notebook_reverse_training.ipynb

```diff
@@ -0,0 +1,1292 @@
+{
```

Comment:
<p>The gdrive_path should probably contain "reverse" as well, or have src and trg flipped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
flipped src and trg